sorc/grid_tools.fd/regional_esg_grid.fd/pmat.f90

```diff
@@ -630,25 +630,21 @@ subroutine cldumf(a,ipiv,d,ff)!                                         [ldum]
 enddo
 end subroutine cldumf
 
-!=============================================================================
+!> @brief SUBROUTINE UDLMM.
```

Comment:
Do not use brief tags with subroutines functions.

Reply:
done